app/models/post.rb

```diff
@@ -1,5 +1,14 @@
 class Post < ApplicationRecord
+  # belongs_to :user, class_name: 'user'
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
   belongs_to :user, class_name: 'User'
-  has_many :comments
-  has_many :likes
+
+  def update_user_post
```

Comment:
- [ ] [OPTIONAL] I recommend renaming your method to something more descriptive like `update_post_counter`

Reply:
Dear Ismailco,

Ok Noted